recipes/simdjson/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            self.options.remove("fPIC")
+        if self.settings.compiler.cppstd and \
+           not self.settings.compiler.cppstd in self._supported_cppstd:
+          raise ConanInvalidConfiguration("This library requires c++17 standard or higher."
+                                          " {} required."
+                                          .format(self.settings.compiler.cppstd))
+
+        if not self._has_support_for_cpp17():
+            raise ConanInvalidConfiguration("This library requires C++17 or higher support standard."
+                                            " {} {} is not supported."
+                                            .format(self.settings.compiler, self.settings.compiler.version))
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+        # In version 0.2.1 CMAKE_CXX_FLAGS are ignored
+        tools.replace_in_file(os.path.join(self._source_subfolder, 'tools', 'cmake', 'FindOptions.cmake'),
+                              'set(CMAKE_CXX_FLAGS "${CXXSTD_FLAGS}',
+                              'set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CXXSTD_FLAGS}',
+                              strict=False)
+
+        # Generating export files by CMake via __export_def (enabled by property WINDOWS_EXPORT_ALL_SYMBOLS)
+        # does not work with whole program optimization.
+        # So disable INTERPROCEDURAL_OPTIMIZATION
+        if self.settings.compiler == "Visual Studio" and self.options.shared:
+            tools.replace_in_file(os.path.join(self._source_subfolder, 'CMakeLists.txt'),
+                                  'set(CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)',
+                                  'set(CMAKE_INTERPROCEDURAL_OPTIMIZATION FALSE)')
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions['SIMDJSON_BUILD_STATIC'] = not self.options.shared
+        cmake.definitions['SIMDJSON_ENABLE_THREADS'] = self.options.threads
+        cmake.definitions['SIMDJSON_DISABLE_AVX'] = not self.options.avx
+        cmake.definitions['SIMDJSON_SANITIZE'] = False
+        cmake.definitions['ENABLE_FUZZING'] = False
+        cmake.configure()
+        return cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        # remove unneeded directories
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
+
+        self.copy("license", src=self._source_subfolder, dst="licenses", ignore_case=True, keep_path=False)
+
+    def package_info(self):
+        self.cpp_info.libs = ['simdjson']
```

Comment:
```suggestion
        self.cpp_info.libs = ['simdjson']
        if self.settings.os == "Linux":
            self.cpp_info.system_libs = ["m"]
```

ldd lib/libsimdjson.so.0
	linux-vdso.so.1 (0x00007ffc82f45000)
	libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x00007f80dd6a6000)
	libm.so.6 => /usr/lib/libm.so.6 (0x00007f80dd560000)
	libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007f80dd546000)
	libc.so.6 => /usr/lib/libc.so.6 (0x00007f80dd37f000)
	/usr/lib64/ld-linux-x86-64.so.2 (0x00007f80dd8e8000)